tests/gocase/unit/type/bitmap/bitmap_test.go

```diff
@@ -0,0 +1,238 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package bitmap
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/go-redis/redis/v9"
+	"github.com/stretchr/testify/require"
+)
+
+type BITOP int32
+
+const (
+	AND BITOP = 0
+	OR  BITOP = 1
+	XOR BITOP = 2
+	NOT BITOP = 3
+)
+
+func Set2SetBit(t *testing.T, rdb *redis.Client, ctx context.Context, key string, bs []byte) {
+	buf := bytes.NewBuffer([]byte{})
+	for _, v := range bs {
+		buf.WriteString(fmt.Sprintf("%08b", v))
+	}
+	for index, value := range buf.String() {
+		require.NoError(t, rdb.SetBit(ctx, key, int64(index), int(value)-int('0')).Err())
+	}
+}
+func GetBitmap(t *testing.T, rdb *redis.Client, ctx context.Context, keys ...string) []string {
+	buf := []string{}
+	for _, key := range keys {
+		cmd := rdb.Get(ctx, key)
+		require.NoError(t, cmd.Err())
+		buf = append(buf, cmd.Val())
+	}
+	return buf
+}
+func SimulateBitOp(op BITOP, values ...[]byte) string {
+	maxlen := 0
+	binarayArray := []string{}
```

Comment:
`binary`?

Reply:
> `binary`?

It means 01 string. Name it according to `tcl` test.

Reply:
I mean, it is a typo, binaray -> binary

Reply:
> I mean, it is a typo, binaray -> binary

After you pointed it out, I didn't even find this mistake. Thank you very much for your careful help. I've fixed it